libs/partners/ai21/tests/integration_tests/test_llms.py

```diff
@@ -0,0 +1,109 @@
+"""Test AI21LLM llm."""
+import pytest
+from ai21.models import Penalty
+
+from langchain_ai21.llms import AI21
+
+
+def test_stream() -> None:
+    """Test streaming tokens from OpenAI."""
+    llm = AI21()
+
+    with pytest.raises(NotImplementedError):
+        for token in llm.stream("I'm Pickle Rick"):
+            assert isinstance(token, str)
+
+
+async def test_astream() -> None:
```

Comment:
🧐 ?
We don't support stream but we support astream?

Reply:
No. Removed